webpack.config.js

```diff
@@ -71,4 +57,26 @@ var config = {
   plugins: plugins
 };
 
-module.exports = config;
+const commonjsConfig = {
+  ...config,
+  output: {
+    path: path.resolve("./dist"),
+    filename: "index.js",
+    sourceMapFilename: "index.map",
+    devtoolModuleFilenameTemplate: function(info) {
+      return "file:///" + info.absoluteResourcePath;
+    }
+  }
+};
+
+const umdConfig = {
```

Comment:
running yarn build seems to break here with this error:

/Users/aron/dev/projects/olio/vr/aframe-typescript-redux/webpack.config.js:61
  ...config,
  ^^^
SyntaxError: Unexpected token ...

spread operator not supported?